### PR TITLE
VectorDraw Enhancements

### PR DESCRIPTION
**Pull request by mtyaka**
This set of changes adds ability to:

- set vector label color (`style: {labelColor: 'green', ...}`)
- set length units (`length_units: 'mm'`)
- set length factor (`length_factor: 2.5`)
- set base angle (`base_angle: 20`)

It also fixes `segment_coords` and `segment_angle` checks that didn't always work correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,10 +84,21 @@ These are the supported vector properties:
...
+* `length_units`: Length units to be displayed in the 'Vector
+  Properties' box (eg. `"mm"`). Defaults to no units.
+* `length_factor`: The factor by which to multiply the length before
+  displaying it in the 'Vector Properties' box. Note that the factor
+  is only applied for display purposes and is ignored in grade
+  checks. Defaults to `1`.
+* `base_angle`: The value to subtract from the vector angle before
+  displaying it in the 'Vector Properties' box. Useful for excercises
+  such as 'box on an incline'. Note that the base angle is only
+  subtracted from the actual angle for display porposes and is ignored
```
Comment (e-kolpakov) [inline README.md]: @mtyaka typo: p**o**rposes

**Comment by e-kolpakov** [comment]
@mtyaka :+1:

File: README.md (hunk 1 above)
Comment (mtyaka) [inline README.md]: Thanks, fixed.

**Comment by dseaton** [comment]
@mtyaka @antoviaque At first pass, the length conversion and units work quite well. See example number 15 Single Slit Diffraction in my repo. Also live here:
https://studio.edge.edx.org/container/i4x://DavidsonNext/Ann101/vertical/3f8075cf68b84a4584f95499b1147e82

Thanks for the fast turn around!

**Comment by antoviaque** [comment]
@dseaton Glad to hear! : )

**Pull request merged**